packages/twenty-front/src/modules/object-record/record-index/states/recordIndexKanbanAggregateOperationState.ts

```diff
@@ -0,0 +1,13 @@
+import { AGGREGATE_OPERATIONS } from '@/object-record/record-table/constants/AggregateOperations';
+import { createState } from 'twenty-ui';
+
+export type KanbanAggregateOperation = {
```

Comment:
Should we use Kanban naming or stick with Board everywhere ?

Reply:
mmh good question.. The columns in the db use kanban, following what has been put in place with kanbanFieldMetadataId. I would stick to kanban here to match the name in db. But questionnable, happy to follow your suggestion if you think its better